proto/opamp.proto

```diff
@@ -635,7 +648,13 @@ enum AgentCapabilities {
     AgentCapabilities_ReportsHealth                  = 0x00000800;
     // The Agent will report RemoteConfig status via AgentToServer.remote_config_status field.
     AgentCapabilities_ReportsRemoteConfig            = 0x00001000;
-
+    // The Agent can report heartbeats.
+    // This is specified by the ServerToAgent.OpAMPConnectionSettings.heartbeat_interval_seconds field.
+    // If this capability is true, but the Server does not set a heartbeat_interval_seconds field, the
+    // Agent should use its own configured interval, which by default will be 30s, the Server may not know this
```

Comment:
```suggestion
    // Agent should use its own configured interval, which by default should be 30s, the Server may not know this
```
We say "should" elsewhere, and I think we should leave it to client libraries to determine the default interval.